pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino

```diff
@@ -0,0 +1,275 @@
+  randomSeed(analogRead(randomPin)); 
+  
+  pinMode(micPin,                    INPUT);
+  pinMode(joyStickPin,               INPUT);
+  pinMode(photoResistorPin,          INPUT);
+  pinMode(startSwitch,               INPUT);
+  pinMode(speakerPin,                OUTPUT);
+  pinMode(hexDisplayPin,             OUTPUT);
+
+  // I2C LCD INIT
+  lcd.init();
+  // Print a message to the LCD.
+  lcd.backlight();
+  lcd.setCursor(0,0);
+  lcd.print(welcomeMessage);
+  lcd.setCursor(2,1);
+  lcd.print(startMessage);
+}
+
+/*
+ * Method to hold the functionality of the panthers_nose command-action.
+ */
+void panthers_nose()
+{
+  currTime = millis();
+  
+  startPlayback(panthersNoseSample, sizeof(panthersNoseSample));
+     
+  displayMessage(panthersNoseCommand); // Displays command on LCD
+     
+  if (joyStickPin == HIGH and millis() - currTime < timeInterval)
+  {
+    userScore += 1; 
+  }
+  else
+  {
+    gameOver(loosingMessage);
+  }
+}
+
+/*
+ * Method to hold the functionality of the victory_lights command-action.
+ */
+void victory_lights()
+{
+  currTime = millis();
+  
+  startPlayback(victoryLightsSample, sizeof(victoryLightsSample));
+     
+  displayMessage(victoryLightsCommand); // Displays command on LCD
+  
+  if (photoResistorPin == HIGH and millis() - currTime < timeInterval)
+  {
+    userScore += 1; 
+  }
+  else
+  {
+    gameOver(loosingMessage);
+  }
+}
+
+/*
+ * Method to hold the functionality of the hail_to_pitt command-action.
+ */
+void hail_to_pitt()
+{
+  currTime = millis();
+  
+  startPlayback(hailToPittSample, sizeof(hailToPittSample));
+     
+  displayMessage(hailToPittCommand); // Displays command on LCD
+     
+  if (readFromMicrophone() and millis() - currTime < timeInterval)
+  {
+    userScore += 1; 
+  }
+  else
+  {
+    gameOver(loosingMessage);
+  }
+}
+
+/*
+ * Method to hold functionality that is executed when the game ends
+ */
+void gameOver(String pMessage)
+{
+  game_started = false;
+  displayMessage(pMessage);
+  displayScore();
+}
+
+/*
+ * Method to hold functionality that chooses a command at random from a list of
+ * predefined commands.
+ */
+int pickRandomCommand()
+{
+  int random_command = random(3);  
+  command_count += 1;
+  return random_command;
+}
+
+/*
+ * Method to hold functionality that is executed when an application error is raised
+ */
+void raise_application_error()
+{
+  Serial.print("Application Error Encountered");
+}
+
+/*
+ * Method to hold the functionality to play the 'Hail to Pitt' song once the game has been won
+ */
+void playWinningJingle()
+{
+
+}
+
+/*
+ * Method to hold the functionality to display the current score
+ */
+void displayScore()
+{
+
+}
+
+/*
+ * Method to hold the functionality to display current commands and game messages on the LCD display
+ */
+void displayMessage(String pMessage)
+{
+
+}
+
+/*
+ * Method to hold the functionality to read an analog value from the microphone
+ */
+boolean readFromMicrophone()

```

Comment:
@cmg137 can you add a link to where you found this online in the README file under Arduino citations & references section?